memory/spring-ai-model-chat-memory-jdbc/pom.xml

```diff
@@ -46,8 +46,8 @@
 		</dependency>
 
 		<dependency>
-			<groupId>org.springframework</groupId>
-			<artifactId>spring-jdbc</artifactId>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jdbc</artifactId>
```

Comment:
Use `spring-data-jdbc` instead of the starter.

Reply:
I will have to change org.springframework.boot.jdbc.DatabaseDriver then

Reply:
@xchopin btw, it is worth considering using `spring-data-jdbc` repositories directly instead of manual query creations at this point. I added a comment to the PR.

Reply:
@leijendary is it still worth it to move to spring-data-jdbc with this incoming PR? 

https://github.com/spring-projects/spring-ai/pull/2803/files

Reply:
@xchopin Yes it is still worth it. The implementation in that PR is the same as the current implementation wherein it uses driver specific syntax. We should probably wait for that PR to be merged though as it is a conflict.